samples/04_gis_analysts_data_scientists/visualize_monthly_changes_in_Hirakund_reservoir_using_video.ipynb

```diff
@@ -0,0 +1,384 @@
+{
```

Comment:
<p>Since we don't typically have dependencies on <code>imageio</code>  and <code>PIL</code>,  add conda or pip install instructions before this cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>